Fix ordering of workshops

pbanaszkiewicz opened the pull request:
Workshops saved from AMY are always sorted by start date in ascending
order (oldest first).

The templates expected different format (usually newest first), so
they were flipped.

pbanaszkiewicz committed:
Fix ordering of workshops

Workshops saved from AMY are always sorted by start date in ascending
order (oldest first).

The templates expected different format (usually newest first), so
they were flipped.

gvwilson commented on bin/get-amy.py:
```diff
@@ -70,6 +70,9 @@ def main(amy_url, output_file, tags, all_=True):
         del workshops
 
+    # Always sort workshops by the ascending start date.
+    unique_workshops = sorted(unique_workshops, key=lambda w: w['start'])
```
And if the start dates are the same, should we sort by something else to get a completely deterministic order?

pbanaszkiewicz replied:
Personally I wouldn't worry about it, but if you want I can add sorting by the slug as well.

gvwilson replied:
Yes please.

On 2016-01-19 10:02 AM, Piotr Banaszkiewicz wrote:

> In bin/get-amy.py 
> https://github.com/swcarpentry/website/pull/183#discussion_r50121970:
> 
> > @@ -70,6 +70,9 @@ def main(amy_url, output_file, tags, all_=True):
> >          del slugs
> >          del workshops
> > -    # Always sort workshops by the ascending start date.
> > -    unique_workshops = sorted(unique_workshops, key=lambda w: w['start'])
> 
> Personally I wouldn't worry about it, but if you want I can add 
> sorting by the slug as well.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/swcarpentry/website/pull/183/files#r50121970.

## 

Dr Greg Wilson
Director of Instructor Training
Software Carpentry Foundation

pbanaszkiewicz committed:
bin/get-amy.py: additionally sort events by slug too

Originally workshops were sorted by start date only, now they're sorted
by slug as well.
